recipes/sail/all/conanfile.py

```diff
@@ -0,0 +1,132 @@
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "src"
+
+    @property
+    def _build_subfolder(self):
+        return "build"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("giflib/5.2.1")
+        self.requires("jasper/2.0.33")
+        self.requires("libjpeg/9d")
+        self.requires("libpng/1.6.37")
+        self.requires("libtiff/4.3.0")
+        self.requires("libwebp/1.2.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.definitions["SAIL_BUILD_APPS"]     = False
+        self._cmake.definitions["SAIL_BUILD_EXAMPLES"] = False
+        self._cmake.definitions["SAIL_BUILD_TESTS"]    = False
+        self._cmake.definitions["SAIL_COMBINE_CODECS"] = True
+        self._cmake.definitions["SAIL_INSTALL_PDB"]    = False
+        self._cmake.definitions["SAIL_THREAD_SAFE"]    = self.options.thread_safe
+        self._cmake.configure(build_folder=self._build_subfolder)
+
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE.txt",       src=self._source_subfolder, dst="licenses")
+        self.copy("LICENSE.INIH.txt",  src=self._source_subfolder, dst="licenses")
+        self.copy("LICENSE.MUNIT.txt", src=self._source_subfolder, dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+        # Remove CMake and pkg-config rules
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        # Move icons
+        rename(self, os.path.join(self.package_folder, "share"),
+                     os.path.join(self.package_folder, "res"))
+
+    def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "Sail")
+
+        self.cpp_info.filenames["cmake_find_package"]       = "Sail"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "Sail"
+        self.cpp_info.names["cmake_find_package"]           = "SAIL"
+        self.cpp_info.names["cmake_find_package_multi"]     = "SAIL"
+
+        self.cpp_info.components["sail-common"].set_property("cmake_target_name", "SAIL::SailCommon")
+        self.cpp_info.components["sail-common"].names["cmake_find_package"]       = "SailCommon"
+        self.cpp_info.components["sail-common"].names["cmake_find_package_multi"] = "SailCommon"
+        self.cpp_info.components["sail-common"].includedirs = ["include/sail"]
```

Comment:
are you sure?

Reply:
Yeah, there is an extra include subdir. So <sail/sail.h> should actually include `include/sail/sail/sail.h`